src/Pages/Channels/M3u8.php

```diff
@@ -72,7 +72,7 @@ public function __invoke(ServerRequestInterface $request)
         }
 
         foreach ($channels as $channelDescriptor) {
-            $content .= "#EXTINF:-1 tvg-id=\"{$channelDescriptor['SERVICE_ID']}\",{$channelDescriptor['NAME']}\r\n";
+            $content .= "#EXTINF:-1 tvg-id=\"{$channelDescriptor['SERVICE_ID']}\",{$channelDescriptor['NAME']->replace("'","")}\r\n";
```

Comment:
`'` should be supported. AFAIK m3u8 are utf-8 encoded.

See https://fr.wikipedia.org/wiki/M3U (french) with `'`examples...

Can you tell which m3u8 software do you use?

Reply:
Yes, I agree. It is unnecessary to remove quotes in the m3u8 file generator.